src/Compiler/Service/FSharpCheckerResults.fs

```diff
@@ -1968,12 +1968,31 @@ type internal TypeCheckInfo
                     match declItemsOpt with
                     | None -> emptyToolTip
                     | Some (items, denv, _, m) ->
-                        ToolTipText(
-                            items
-                            |> List.map (fun x ->
-                                let symbol = Some(FSharpSymbol.Create(cenv, x.Item))
-                                FormatStructuredDescriptionOfItem false infoReader tcAccessRights m denv x.ItemWithInst symbol width)
-                        ))
+                        match items with
```

Comment:
As there are now (potentially) multiple symbols, we use the property for the tooltip if possible.